javascript/javascript1/week1/goodboy-oldboy.js

```diff
@@ -0,0 +1,13 @@
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+
+let dogYear = dogYearFuture - dogYearOfBirth;
+
+// 1st case showing in human years
+let shouldShowResultInDogYears = false;
```

Comment:
You have to use 'conditionals' to either run one or the other command.

quick intro to the syntax of conditionals:
```js
if (condition){
// execute this code
}else{
 // execute some other code
}
```

Reply:
Thanks! Fixed it!